imports/api/database-controller/focus-area/methods.js

```diff
@@ -77,6 +79,26 @@ export const createModuleListObject = function createNewListOfModuleObject(modul
   return moduleToBeStored;
 }
 
+export const getFocusAreaRequirement = function getFocusAreaRequirement(getFocusRequirementIDArray)  {
```

Comment:
Because the grad requirements are rather complex, I kinda like your way of programming as it is very verbose and describes the entire process through the code rather than me having to read some docs somewhere. :)